### ISSUE TITLE
A11y_ML .NET Model Builder_ Data Preview_ Keyboard: Keyboard tab stop is provided to every non-interactive cell in the table which is not necessary

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1131]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #Feature-ML.NETModelBuilder-Win32-Nov2020; #A11ySev2; #NonBenchmark; #Visual Studio; #Keyboard; #WCAG1.3.1;

### Environment Details:
Microsoft Visual Studio 2019 Preview Version 16.8.0 Preview 6.0

### Note:
Same issue is observed all the scenarios where tables are present

### Repro Steps:

1. Open Visual Studio Enterprise 2019 Preview 
2. Create a new .Net Core Console App (C#) 
3. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
4. Tab till "Text Classification" scenario and select the scenario
5. Tab till "Next" button and press enter
6. Select file as input data source and Select the file that has your data
7. Tab till the grid and verify whether tab stop is provided for non interactive content

### Actual Result:
Keyboard tab stop is provided to every non-interactive cell in the table which is not necessary

### Expected Result:
keyboard tab stop should not be provided for non-interactive content

### User Impact:
keyboard only users will get confused if tab stop is provided for non-interactive content

### Attachment for Reference

[1131_ML .NET Model Builder_ tab stop is provided for non interactive table cells.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5635950/1131_ML.NET.Model.Builder_.tab.stop.is.provided.for.non.interactive.table.cells.pptx)

**Comment by SaisreeDandamudi** [comment]
[1131_ML.NET.Model.Builder_tab stops is removed for sortable column headers as well.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/6041911/1131_ML.NET.Model.Builder_tab.stops.is.removed.for.sortable.column.headers.as.well.pptx)

@beccamc Verified the bug in the latest build and now tab stop is removed for entire table. Even for the column headers
But the column headers in the table are sortable. Tab stop should be available for sortable column headers
Please find the above attachment for reference

**Comment by beccamc** [comment]
These columns actually should not be sortable. I'm removing that functionality. Will post build when available.

**Comment by beccamc** [comment]
Build https://devdiv.visualstudio.com/_apis/resources/Containers/7343454/VSSetup?itemPath=VSSetup%2FMicrosoft.ML.ModelBuilder.vsix

**Comment by SaisreeDandamudi** [comment]
Verified the bug in latest build and the issue is fixed. Hence closing the bug
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.10.0 Preview 1.0

**Comment by poojapatil36** [comment]
Github tag: #Closed;
